<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="OGLWidget" name="glwidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Toolbox</string>
      </property>
      <widget class="QDial" name="dial_Light">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
      <widget class="QDial" name="dial_Zoom">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>20</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Light</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Zoom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>180</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Rotation</string>
       </property>
      </widget>
      <widget class="QDial" name="xRot">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>210</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
      <widget class="QDial" name="yRot">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>270</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
      <widget class="QDial" name="zRot">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>330</y>
         <width>50</width>
         <height>64</height>
        </rect>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>695</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>oglwidget.h</header>
   <slots>
    <slot>setParamA(double)</slot>
    <slot>setParamB(double)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
